controller/node_controller.go

```diff
@@ -47,6 +47,8 @@ const (
 
 	unknownDiskID = "UNKNOWN_DISKID"
 
+	kernelConfigFilePathPrefix = "/boot/config-"
```

Comment:
Is the config file existing in the pod?

Reply:
No, at least not in the `longhorn-manager` pod.

Reply:
>Should we mount /boot on the host to /host/boot/ in the pod?

If we don't want to use the binary commands to check if the configure file exists and the module is enabled in the configure file, yes.